roles/neutron-common/meta/main.yml

```diff
@@ -5,3 +5,9 @@ dependencies:
     service: neutron
     logdata: "{{ neutron.logs }}"
   - role: openstack-source
+    project_name: neutron
+    project_rev: "{{ neutron.source.rev }}"
+    python_requirements:
+      - { name: alembic, version: "{{ neutron.alembic_version }}" }
```

Comment:
Looks like if we pull at least https://git.openstack.org/cgit/openstack/neutron/commit/?id=d7ee2ff991e60ad54ff2125a397e9293ba966549 from stable/juno, we should be clear of the reason for pinning alembic. Worth testing out I suppose.